### PR TITLE
Stock feature

### PR DESCRIPTION
**Pull request by sureshhemal**

### CODE REVIEW

File: resources/js/Components/Stock/Category/Form.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            <FormGroup>
+                <Label for="uuid"> uuid </Label>
+                <WebsolInput
+                    id="uuid"
+                    type="text"
+                    v-model="form.uuid"
+                    :readonly="isEditing"
+                />
+                <div
+                    v-if="form.errors.uuid"
```
Comment (sureshhemal) [inline resources/js/Components/Stock/Category/Form.vue:13]: @Kushan1994 errors not showing.

File: resources/js/Components/Stock/Product/Form.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      <option
+        v-for="category in props.subCategories"
+        :key="category.id"
+        :value="category.id"
+      >
+        {{ category.uuid }} -- {{ category.name }}
+      </option>
+    </WebsolDropdown>
+    <div
+      v-if="form.errors.name"
```
Comment (sureshhemal) [inline resources/js/Components/Stock/Product/Form.vue:15]: @Kushan1994 errors are not visible on frontend

File: resources/js/Components/Stock/SubCategory/Form.vue [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            <FormGroup>
+                <Label for="uuid"> uuid </Label>
+                <WebsolInput
+                    id="uuid"
+                    type="text"
+                    v-model="form.uuid"
+                    :readonly="isEditing"
+                />
+                <div
+                    v-if="form.errors.uuid"
```
Comment (sureshhemal) [inline resources/js/Components/Stock/SubCategory/Form.vue:30]: @Kushan1994 errors are nit visible

File: resources/js/Components/Supplier/Form.vue [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+            class="text-red-500 text-xs mt-1"
+            v-text="form.errors.name"
+        ></div>
+    </FormGroup>
+
+    <FormGroup>
+        <Label for="address"> Address </Label>
+        <WebsolInput v-model="form.address" id="address" type="text" />
+        <div
+            v-if="form.errors.address"
```
Comment (sureshhemal) [inline resources/js/Components/Supplier/Form.vue:16]: @Kushan1994 errors are not visible

File: resources/js/Components/LayoutComponents/Navbar.vue [hunk 5]
```diff
@@ -4,11 +4,13 @@
         <div class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
             <div class="flex justify-between h-16">
                 <div class="flex">
-                    <div class="hidden sm:ml-6 sm:flex sm:space-x-8"></div>
+                    <div class="hidden sm:ml-6 sm:flex sm:space-x-8">
+                        {{ $page.component.split('/')[0] }}
```
Comment (sureshhemal) [inline resources/js/Components/LayoutComponents/Navbar.vue:8]: @Kushan1994 reformat this as page header. 

`<h1>`
`color = blue`

**Review by Kushan1994 — APPROVED** [review]

**Pull request merged**